src/hardware/vga_draw.cpp

```diff
@@ -148,15 +148,15 @@ static uint8_t byte_clamp(int v)
 	return v < 0 ? 0u : (v > 255 ? 255u : static_cast<uint8_t>(v));
 }
 
-static uint8_t *Composite_Process(uint8_t border, uint32_t blocks, bool doublewidth)
+static uint8_t *Composite_Process(uint8_t border, uint32_t blocks, bool double_width)
 {
 	static int temp[SCALER_MAXWIDTH + 10] = {0};
 	static int atemp[SCALER_MAXWIDTH + 2] = {0};
 	static int btemp[SCALER_MAXWIDTH + 2] = {0};
 
 	int w = blocks * 4;
 
-	if (doublewidth) {
+	if (double_width) {
```

Comment:
Not comment on the code change - just the commit message, suggest being a bit more descriptive, `"Rename a VGA function variable"` instead of `Cleanup`, cause it can get a bit ambiguous reading history:

![2023-08-31_18-20](https://github.com/dosbox-staging/dosbox-staging/assets/1557255/b4df734a-1fc6-40d7-8e78-bfece05f81b9)

Reply:
Sure, I always struggle with those, especially if lots of cleanup stuff gets bundled together 😄 Changing this particular one.